src/Pagerfanta/Pagerfanta.php

```diff
@@ -105,6 +105,11 @@ public function getNormalizeOutOfRangePages()
         return $this->normalizeOutOfRangePages;
     }
 
+    /**
```

Comment:
I'm -1 to use PHPDoc unless it helps for something.

For me documenting is helpful to know things you can't guess.

Documenting private functions doesn't help much.
Documenting variable names which are some lines below doesn't help much, overall with good variable names which already indicate what they have.
Documenting "mixed" returns don't help much.